.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
This is probably a directory you want to add to a .gitignore file :)

Reply:
done!